src/main/java/net/dries007/tfc/objects/te/TETickCounter.java

```diff
@@ -25,12 +25,14 @@ public long getTicksSinceUpdate()
     public void resetCounter()
     {
         lastUpdateTick = CalendarTFC.PLAYER_TIME.getTicks();
+        markForSync();
         markDirty();
```

Comment:
`markForSync` already calls `markDirty`, no need to call both here. Same as below